2016/proposals/maurya-kumar-raj.md

```diff
@@ -0,0 +1,71 @@
+
+This project would build a system for sending a species name to one or more of the taxonomic name resolution services and determine the best name to appear in the generated databases. The project is composed of the following main components: 
+* Accessing the resolution services.
+* Determining the selection strategy.
+* Updating the data model. 
+* The user interface and the control flow.
+
+The download operation is performed through Engine class, which uses urllib library to fetch the dataset file, and store it locally. This operation happens inside download_file() function in ./lib/engine.py file.
+
+The general problem that we will be dealing would be to maintain the accuracy of datasets for which the names of species are redefined that raises the difficulty in combining the multiple datasets together.
+
+## Schedule of Deliverables
+
+This time line has been made keeping in mind that it will require work at least 40 hours per week. I am not planning on taking any vacations during the summer and would dedicatedly work on the project and also, I will continue to contribute afterwards.
+
+### May 25th -  June 7th
+
+### June 8th - June 21th
+
+### June 22nd - July 5th
+
+### July 6th - July 19th
+
+### July 20th - August 2nd
+
+### August 3rd - August 16th
+
+### August 17th - August 21th 19:00 UTC
+
+## Future works
+
+* I will continue to work on the Retriever Project and finish some feature requests and i will be involved in fixing some bugs in Retriever.
+* Adding more reliable name resolution APIs.
+* Improving the accuracy of the best name deciding algorithm.
+* Bug fixing the existing and adding more engines for different formats of data. For example - JSON, XML etc.
+
+## Open Source Development Experience
+
+I have worked on few open source projects like Google's tensorflow that deals with Deep Learning and involves the Machine Learning concepts. I also have contribution in fixing bugs in Mozilla community at bugzilla and also contributed as Firefox Student Ambassador.
+
+I have also worked with a start-up from IIT Kanpur,SIDBI Incubation Center, (India) which involved the development of Dynamic UI graph using plot.ly API for Python and Java Script for testing and plotting the large experimental data sets.
+
+I have been the member of NumFocus community and supporting it as a volunteer.
+
+## Academic Experience
+
+I am currently a 3rd year (B.Tech) Computer Science and Engineering student at University Institute of Engineering and Technology, Kanpur, India. I have done some academic projects on database management systems. I am a FOSS enthusiast and like to contribute, collaborate for the same.
+
+This project deals with implementation and automating the process of accessing the data in EcoData Retriever that requires the knowledge Python and database systems like MySql, SqLite, which I am familiar with and have experience as well as done few projects on that technology.
```

Comment:
\[some thing like this\]
This project deals with implementing and automating the process of accessing the data with the taxonomic names resolved.
The projects requires the knowledge of Python and database systems like MySql, SqLite.
I have worked on some/similar/familiar projects that had the same requirements and skill set